packages/wasm/src/index.ts

```diff
@@ -0,0 +1,73 @@
+import { Event, EventProcessor, Hub, Integration, StackFrame } from '@sentry/types';
+
+import { patchWebAssembly } from './patchWebAssembly';
+import { getImage, getImages } from './registry';
+
+/** plz don't */
+function patchFrames(frames: Array<StackFrame>): boolean {
+  let haveWasm = false;
+  frames.forEach(frame => {
+    if (!frame.filename) {
+      return;
+    }
+    const match = frame.filename.match(/^(.*?):wasm-function\[\d+\]:(0x[a-fA-F0-9]+)$/);
```

Comment:
On a recent chrome version, this `match[1]` corresponds to a wasm url schema, like `wasm://wasm/0245f4e2`. It seems chrome assigns some virtual URL to the binary, with some hash. But in the patch-web-assembly code, it uses the url from the fetch promise `registerModule(..., response.url);`, which is the actual url where I fetched the wasm binary from. So getImage never ends up finding a match.

I can file a separate issue if that's helpful, but I'm wondering if anyone knows where these wasm url schemes came from, and how we can match that hash up with the wasm binary we fetched + compiled?

Reply:
Here's a simple open source example, but this doesn't seem limited to this package, it seems to effect most/all wasm binaries, at least on this version of chrome

http://nodeca.github.io/pica/demo/
![image](https://user-images.githubusercontent.com/25588019/143978213-0be24901-cf3c-4e12-a9ce-29a4a82e5fa2.png)